spec.bs

```diff
@@ -124,36 +147,6 @@ spec: permissions-policy; urlPrefix: https://www.w3.org/TR/permissions-policy/
 spec: attestation; urlPrefix: https://github.com/privacysandbox/attestation
     type: dfn
         text: enrolled
-spec: private-aggregation-api; urlPrefix: https://patcg-individual-drafts.github.io/private-aggregation-api/
-    type: dfn
-        text: Private Aggregation; url:
```

Comment:
This little tidbit is what enabled us to link to the Private Aggregation spec with `[=Private Aggregation=]`. It's kind of a hack though -- it works by pretending there's a dfn with that name, then overriding the value of the URL hash as the empty string. Technically, it kind of got the job done, but I suppose it risks colliding with something that Private Aggregation actually wanted to export.

Reply:
Definitely a hack, but I don't think there's a real risk of collision here.

Reply:
Specifically, I was thinking that mixing and matching link-defaults and anchors might cause occurrences of `[=Private Aggregation=]` to incorrectly autolink to the permissions policy string, declared as [`<dfn>private-aggregation</dfn>`](https://github.com/patcg-individual-drafts/private-aggregation-api/blob/3433a3e8ff358e3b93a3b57b31ce63204bf8afb4/spec.bs#L494), either now or in a future version of bikeshed.